argcomplete/_check_console_script.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
It would be nice to add a docstring here describing what this file does and linking to the definition and docs for console scripts.

Reply:
Addressed in #263